tests/Unit/ControlSystem/Test_Tags.cpp

```diff
@@ -29,7 +31,46 @@ struct LabelA {};
 using system = control_system::TestHelpers::System<
     2, LabelA, control_system::TestHelpers::Measurement<LabelA>>;
 
-struct MetavarsEmpty {};
+struct MetavarsEmpty {
+  static constexpr size_t volume_dim = 3;
+};
+
+class FakeCreator : public DomainCreator<3> {
```

Comment:
Can this test use the `FakeCreator` class added in the previous commit?  They look the same except the other one has option-parsing stuff.